src/index.ts

```diff
@@ -113,26 +114,57 @@ const ts2gas = (source: string) => {
   );
   // output = output.replace(/^\s*exports\[\"default\"\].*$\n/mg, '');
 
-  // Transpile
-  // https://www.typescriptlang.org/docs/handbook/compiler-options.html
-  const result = ts.transpileModule(source, {
+  /** These settings can be overriden */
+  const defaults: ts.TranspileOptions = {
     compilerOptions: {
-      lib: ['ES2015'],
-      target: ts.ScriptTarget.ES3,
       noImplicitUseStrict: true,
-      noLib: true,
       experimentalDecorators: true,
+      // pretty: true,
+    },
+  };
+
+  /** These the settings are always used */
+  const statics: ts.TranspileOptions = {
+    compilerOptions: {
+      target: ts.ScriptTarget.ES3,
+      noLib: true,
+      // lib: [  // Not be relevant with ts.transpileModule(). also 'noLib' is true
```

Comment:
We don't need this comment.

Reply:
Gone (but I added one about renamedDependencies for documentation/reference)